### PR TITLE
Update @param annotations in InputInterface

### PR DESCRIPTION
**Pull request by sigma-z**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

Just added some missing @param doc block annotations in the InputInterface (component: symfony/console)

### CODE REVIEW

File: src/Symfony/Component/Console/Input/InputInterface.php [hunk 1]
```diff
@@ -148,6 +154,8 @@ public function isInteractive();
 
     /**
      * Sets the input interactivity.
+     * 
+     * @param bool $interactive The interactive flag
```
Comment (OskarStark) [inline src/Symfony/Component/Console/Input/InputInterface.php]: These are not very useful, that’s why they are not here

File: src/Symfony/Component/Console/Input/InputInterface.php [hunk 2]
```diff
@@ -117,7 +120,7 @@ public function getOptions();
     /**
      * Returns the option value for a given option name.
      *
-     * @return string|string[]|bool|null The option value
+     * @return string $name The option name
```
Comment (OskarStark) [inline src/Symfony/Component/Console/Input/InputInterface.php]: This change looks wrong

**Comment by nicolas-grekas** [comment]
Hello, thanks for the PR but I'm going to close it as it's wrong: we don't add docblocks that don't add to existing signatures, and changing return types would change the meaning of the interfaces.

**Comment by sigma-z** [comment]
I thought it is an inconsistency that the methods setArgument() and setOptions() does not mention the @param annotation for the argument $name, but I can see now, that in your opinion the signature provides all the information you need.
Sorry for disturbing.

**Pull request closed without merging**